app/assets/stylesheets/users.scss

```diff
@@ -0,0 +1,95 @@
+// Place all the styles related to the users controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+
+.signin-form{
+  .row { margin-right: 0px !important; margin-left: 0px !important;}
```

Comment:
Do you guys have some code conventions? How about make it all consistent with rules like here: https://sass-guidelin.es/#syntax--formatting or any other guideline?